PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -24,7 +24,7 @@ public static void Main(string[] args)
 
         public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
         {
-            return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+            return (((float)numberCorrectAnswers / (float)numberOfQuestions * 100)).ToString("n0") + "%";
```

Comment:
Not bad, I would have actually done something like

```suggestion
return Math.Round((float)numberCorrectAnswers / numberOfQuestions * 100) + "%";
```

Only need to convert one to a float since it till implicitly do float math from there on out.